src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -16,6 +16,16 @@ export interface AdminVenuesProps {
 }
 
 export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const [showCreateModal, setShowCreateModal] = useState<boolean>(false);
+
+  const showModal = useCallback(() => {
+    setShowCreateModal(true);
+  }, []);
+
+  const hideModal = useCallback(() => {
+    setShowCreateModal(false);
+  }, []);
```

Comment:
This is a pattern we tend to use all over the codebase. We could extract it into a `useModalControls` or similar custom hook, and then just use that instead of having to set up this boilerplate all the time.